PayPalWebPayments/src/main/java/com/paypal/android/paypalwebpayments/PayPalWebCheckoutClient.kt

```diff
@@ -57,8 +62,11 @@ class PayPalWebCheckoutClient internal constructor(
      */
     fun start(request: PayPalWebCheckoutRequest) {
         analyticsService.sendAnalyticsEvent("paypal-web-payments:started", request.orderId)
-        payPalWebLauncher.launchPayPalWebCheckout(activity, request)?.let { launchError ->
-            notifyWebCheckoutFailure(launchError, request.orderId)
+
+        activityReference.get()?.let { activity ->
+            payPalWebLauncher.launchPayPalWebCheckout(activity, request)?.let { launchError ->
+                notifyWebCheckoutFailure(launchError, request.orderId)
+            }
```

Comment:
I think we should return an error if activity we can't get the activity reference - PayPal won't be launched but merchant wont get any notification (same for vault)